### DISCUSSION TITLE
Default widgets

### DISCUSSION THREAD

**Discussion by justsee** [vendurehq/vendure#1820] [Q&A] [answered]
Out-of-the-box Vendure comes with pre-existing widgets.

How can a fresh install be configured so none of those widgets are available?

The documentation is unclear, and reading the source it didn't seem straightforward to undo [this configuration](https://github.com/vendure-ecommerce/vendure/blob/master/packages/admin-ui/src/lib/dashboard/src/default-widgets.ts) on a local install.

**Comment by michaelbromley** [comment]
Hi,
It's currently not possible to omit those default widgets. You can however omit them from the default layout, as covered here: https://www.vendure.io/docs/plugins/extending-the-admin-ui/dashboard-widgets/#setting-the-default-widget-layout

**Comment by justsee** [comment]
Hi,

Thanks for the feedback (and for the project!).

Ah okay. So every role with `read order` permissions can add a widget with total daily / weekly / monthly sales for a business?

Is there a way of setting the default layout *and* removing the ability for any user to add widgets to their own dashboard?

Otherwise a basic sales role (to review and fulfill individual orders) also implicitly has a higher-level financials role (ability to see detailed financials for a business).

**Reply by michaelbromley** [reply]
This is a good point and as mentioned above, it is _not_ currently possible to omit the default widgets. I will convert this issue into a feature request.

**Reply by michaelbromley** [reply]
See: https://github.com/vendure-ecommerce/vendure/issues/1832

**Comment by michaelbromley** [comment, marked as answer]
This feature was added in v1.8.0
